GTBitsOfGood/gt-scheduler-website
# Rohan/2 search pane course catalog

## Add view for search column in Course Search tab

Issue Number(s): #2 

What does this PR change and why?
Add the draft UI of course search in "Course Search" tab (2nd column)

<img width="800" alt="image" src="https://user-images.githubusercontent.com/39681900/155656353-bf326c85-5f3f-4b16-8200-2aafc1914ba3.png">


### Checklist


### Critical Changes

- No critical changes

### Related PRs

- No related PRs

### How to Test

- `yarn start`
- navigate to "Course Search Tab"
- Check if the query div (2nd column) matches as described in [Figma](https://www.figma.com/file/CkystexX4qQO9OnZRxktXa/GT-Scheduler-%2F-Spr21?node-id=18%3A2)
- Check if it works on mobile view, which it should
- Double check if the 2nd column in the "Schedule" tab is working fine, bc we thought we were replacing that column with our new column
